web/router.ex

```diff
@@ -10,7 +10,7 @@ defmodule Castle.Router do
   # end
 
   pipeline :api do
-    plug :accepts, ["json"]
+    plug :accepts, ["json", :hal]
```

Comment:
does this need to be `"hal"`? Seeing 406 error: `(Phoenix.NotAcceptableError) no supported media type in accept header, expected one of ["json", :hal]`